pkgs/tools/security/bitwarden/default.nix

```diff
@@ -128,6 +91,41 @@ buildNpmPackage' {
     popd
   '';
 
+  doCheck = true;
+
+  nativeCheckInputs = [
+    dbus
+    (gnome.gnome-keyring.override { useWrappedDaemon = false; })
+  ];
+
+  checkFlags = [
+    "--skip=password::password::tests::test"
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    pushd ${cargoRoot}
+    export HOME=$(mktemp -d)
+    export -f cargoCheckHook runHook _eval _callImplicitHook
+    export cargoCheckType=release
```

Comment:
This really shouldn't be necessary but without it we run
```
cargo test -j 8 -- --target x86_64-unknown-linux-gnu --frozen -- --test-threads=8 --skip=password::password::tests::test
```
and get `error: Unrecognized option: 'target'`.